render/json.go

```diff
@@ -102,8 +102,7 @@ func (r SecureJSON) Render(w http.ResponseWriter) error {
 	// if the jsonBytes is array values
 	if bytes.HasPrefix(jsonBytes, bytesconv.StringToBytes("[")) && bytes.HasSuffix(jsonBytes,
 		bytesconv.StringToBytes("]")) {
-		_, err = w.Write(bytesconv.StringToBytes(r.Prefix))
-		if err != nil {
+		if _, err = w.Write(bytesconv.StringToBytes(r.Prefix)); err != nil {
```

Comment:
I'd even write:
```suggestion
		if _, err := w.Write(bytesconv.StringToBytes(r.Prefix)); err != nil {
```
here and elsewhere. This way you don't reassign to the outer `err` variable.